newrelic_lambda_cli/templates/license-key-secret.yaml

```diff
@@ -18,14 +18,17 @@ Parameters:
   ViewPolicyExportName:
     Type: String
     Default: NewRelic-ViewLicenseKeyPolicyARN
+  NrAccountId:
+    Type: String
+    Default: NEW_RELIC_ACCOUNT_ID
```

Comment:
We shouldn't have a default if it's not valid. By not providing a default, you'll force the user to provide a value, which is what we want here.